ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml

```diff
@@ -12,6 +12,7 @@ images:
 - dockerfile_literal: |-
     FROM src
     RUN ./hack/dnf_safe -y install python3-openstackclient python3-octaviaclient jq && dnf clean all
+    RUN python -m pip install yq
```

Comment:
It looks like you changed your mind and used `jq` in the end. There's no need to install `yq` in that case.

Reply:
Gotcha!